controllers/administration.js

```diff
@@ -1164,31 +1164,35 @@ const getUsersWithoutConsent = async (req, roleName, classId) => {
     let users = [];
 
     if (classId) {
-        const _class = await api(req).get('/classes/' + classId, {
+        const klass = await api(req).get('/classes/' + classId, {
             qs: {
                 $populate: ['teacherIds', 'userIds'],
             }
         });
-        users = _class.userIds;
-        users = users.map(user => {
-            user.displayName = `${user.firstName} ${user.lastName}`;
-            return user;
-        });
+        users = klass.userIds.concat(klass.teacherIds);
     } else {
         users = (await api(req).get('/users', { qs, $limit: false })).data;
     }
 
-    const consents = (await api(req).get('/consents')).data;
-
-    const usersWithoutConsent = users.filter(user => {
-        let userWithConsent = consents.find(consent => {
-            return consent.userId === user._id;
-        });
+    const consents = (await api(req).get('/consents', {
+        qs: {
+            userId: { $in: users.map(u => u._id) },
+            $populate: 'userId',
+            $limit: false,
```

Comment:
Can you test if $limit=false send from client work? I expect that our global server hooks format the boolean to a string and therefore it do not work.

Reply:
**without $limit**

![limit](https://user-images.githubusercontent.com/5978382/52998918-bb753b00-3424-11e9-93cd-3d545865b863.PNG)
-----------------------
**$limit: false**

![limit_false](https://user-images.githubusercontent.com/5978382/52998926-be702b80-3424-11e9-8eac-7d32e9063d4c.PNG)